### ISSUE TITLE
VS2013 compilation errors, ideas?

### ISSUE THREAD

**Issue by magnetowasnotright** [Herdinger/EmulationStation#47]
**EDIT**: since I found my way with Cmake, I'll update the question.

Do anyone compiling ES in Visual Studio 2013 can answer?
1. Any known issue with VS2013 and the dependencies (Boost 1.60.0, Eigen 3.2.8, FreeImage 3.1.7.0, SDL 2.0.4, cURL 7.47.1, FreeType 2.4.9)? Or should I use older/newer versions?
2. I have, most of the time, errors about unresolved symbols or with msvcprtd.lib. Seems like problems when compiling the libs. Even using MT or MTd, I couldn't make ES compile yet. Any idea?

**Comment by DanielLemes** [comment]
In fact now I made it work, but with errors related to Boost:

> CMake Warning at C:/Users/Daniel/Downloads/cmake-3.6.2-win64-x64/share/cmake-3.6/Modules/FindBoost.cmake:743 (message):
>   Imported targets not available for Boost version 106200
> Call Stack (most recent call first):
>   C:/Users/Daniel/Downloads/cmake-3.6.2-win64-x64/share/cmake-3.6/Modules/FindBoost.cmake:842 (_Boost_COMPONENT_DEPENDENCIES)
>   C:/Users/Daniel/Downloads/cmake-3.6.2-win64-x64/share/cmake-3.6/Modules/FindBoost.cmake:1395 (_Boost_MISSING_DEPENDENCIES)
>   CMakeLists.txt:48 (find_package)

I'll keep trying...

**Comment by zigurana** [comment]
Hey, I am also still struggling with this, only on VS2015.
Did you see #4 ?

**Comment by magnetowasnotright** [comment]
Yes, I followed zimmann suggestion about /MT or /MTd all through the project and libraries, but no luck yet... I spent five days trying everthing I could. Very tired for now, but I'll try again soon.
